pkg/operator/deploy/staticresources/master/deployment.yaml.tmpl

```diff
@@ -49,3 +49,8 @@ spec:
       - key: node-role.kubernetes.io/master
```

Comment:
do we need this now that we will tolerate NoSchedule on its own (matching any key)?

Reply:
We don't actually need the one below.  Good catch.  We still need the key one so we tell it explicitly to run on masters.  This keeps the worker pod on workers and the master pod on masters.

Reply:
We need them both.  One ensures the master pod gets scheduled on the master nodes, the other ensures that if the node is cordoned, it can still schedule on the node.  

Ran a few tests to confirm: 
1. Place anti-affinity rules on worker deployment to not run on the same node
2. Scaled up the number of replicas, it did not schedule on the master nodes

1. Cordoned all nodes `oc adm cordon <node> <...>`
2. Deleted the master pod
3. Master pod wouldn't schedule
4. Added these tolerations to master and worker nodes
5. Master and worker pods scheduled

Reply:
@s-amann is right, we likely don't need this. The nodeSelector will enforce the scheduling, but if we are tolerating NoSchedule for any key, we don't need to also specifically tolerate it for the master taint. That said, it shouldn't have any negative effects to leave all three tolerations.

Reply:
Fixed.